uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/ACL/LeaveApproverTrait.php

```diff
@@ -1,24 +1,28 @@
 <?php
 
-/**
- * Access Control Lists Custom Query generator class
- *
- * This class is basically an Utility class used to generate queries needed for overriding addSelectWhereClause
- * for some specific BAO's
- *
- */
-class CRM_HRLeaveAndAbsences_API_Query_ACLQueryHelper {
+trait CRM_HRLeaveAndAbsences_ACL_LeaveApproverTrait {
```

Comment:
We should probably change the name of this trait also to `CRM_HRLeaveAndAbsences_ACL_LeaveInformationTrait`